packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -0,0 +1,3 @@
+from pkgpanda.util import load_json
+
+dcos_config = load_json('/opt/mesosphere/etc/expanded.config.json')
```

Comment:
OK. I don't have any other suggestion here. I see other pieces of code adopting the same approach of reading from `expanded.config.json` on master.